packages/rule-components/src/ReportDetails/index.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Grid,
+    GridItem,
+    Stack,
+    StackItem,
+    Text,
+    TextVariants,
+    Title
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Battery } from '@redhat-cloud-services/frontend-components/components/Battery';
+import RuleFeedback from './RuleFeedback';
+import './ReportDetails.css';
+import ReactMarkdown from 'react-markdown';
+
+const totalRiskMeta = [
+    {
+        label: 'low',
+        description: 'low severity desc for total risk',
+        IconComponent: Battery
+    },
+    {
+        label: 'moderate',
+        description: 'moderate severity desc for total risk',
+        IconComponent: Battery
+    },
+    {
+        label: 'important',
+        description: 'important severity desc for total risk',
+        IconComponent: Battery
+    },
+    {
+        label: 'critical',
+        description: 'critical severity desc for total risk',
+        IconComponent: Battery
+    }
+];
+
+const riskOfChangeMeta = [
+    {
+        label: 'low',
+        description: 'low severity desc for risk of change',
+        IconComponent: Battery
+    },
+    {
+        label: 'moderate',
+        description: 'moderate severity desc for risk of change',
+        IconComponent: Battery
+    },
+    {
+        label: 'important',
+        description: 'important severity desc for risk of change',
+        IconComponent: Battery
+    },
+    {
+        label: 'critical',
+        description: 'critical severity desc for risk of change',
+        IconComponent: Battery
+    }
+];
+
+const LinkInDetails = ({ children, href }) => (
```

Comment:
Does this really need to be a component? If so I would much rather see it in specific file instead of in `index.js`